src/Umbraco.Cms.Api.Management/Services/NewsDashboard/NewsDashboardOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace Umbraco.Cms.Api.Management.Services.NewsDashboard;
+
+/// <summary>
+/// Options for the news dashboard service.
+/// </summary>
+public class NewsDashboardOptions
```

Comment:
We should make an interface for this so it's easier to replace. Additionally, `Options` is used for the `IOptions`, and this isn't really a setting; it's something that provides a timespan. I think a name like `INewsCacheDurationProvider` would be better.